utils.py

```diff
@@ -498,3 +498,15 @@ def get_template_dir_prefix():
     template_path = ('/templates/head' if feconf.IS_MINIFIED
                      or not feconf.DEV_MODE else '/templates/dev/head')
     return '%s%s' % (get_asset_dir_prefix(), template_path)
+
+
+def convert_unicode_to_str(state_name):
```

Comment:
The arg should not be called `state_name`. This is supposed to be a generic utility class.